### PR TITLE
Fix issues with sub-outs' names + API changes + VR command change + minor fixes

### PR DESCRIPTION
**Pull request by camelwater**
## Fix issues with player names after substitutions
Primary issues:
* Players who are subbed out have their names on tables stored as a string (their Lounge name if they have one). This means name changes have no effect on players who are subbed out (both name changes done before and after the substitution).
* Non-Lounge players have another issue on top of the previous one: their name on tables will contain the "(Tabler Changed)" warning, which shouldn't be there.

**Solution:**
- The "Tabler Changed" warning is now not actually part of the player's name (`Player.Player.name`), but appended to the end of the player's name in the method `Player.Player.get_full_display_name()`. That way a player's "real" name that is shown on tables (without the "Tabler Changed") can be distinguished from the name that is shown elsewhere (with the "Tabler Changed").
- Each `Player.Player` object also stores whether it was subbed out and/or had its name changed--so, calling `Player.get_display_name()` and `Player.get_sub_out_name()` can be used instead of checking if a player is present in the `Room.name_changes` dictionary. 
   - Ideally, each player's `lounge_name` attribute should be usable as well, but I'm not sure if it is due to the lack of use in the current codebase (with `UserDataProcessing.lounge_get()` used instead). I'm sure this isn't too difficult to add though, and this would make determining which name to display much more straightforward (just calling the appropriate Player method because the Player object will figure it out).
- Sub out information is now stored as a dict instead of a list (more readable and easier to manage)
- Sub out mii name and lounge name are no longer stored in `Room.sub_ins` (because they are just strings and will not reflect name changes), so in order to get a sub out's display name, you can call `Room.get_player_from_FC()` and then access their name. 

## API change
- Added `show_team_names` and `sort_teams` optional query parameters for `/api/html/team_scores` endpoint. (They default to True.)
- If `sort_teams` is set to False, it will sort the scores alphabetically instead of by scores.

## VR command update
- VR command has a "Refresh" button attached to the message. When it is clicked the message is edited with the new data from mkwx.

## Other changes
- Added ability to edit a player's score by using `+-`. Behaves the exact same way as normal edits, but may be easier in certain cases (someone wants to quickly correct something and they need to give a player an extra 3 points to their GP2: `/edit player 2 +3`.
- Added `/editrace [raceNumber] [...correct placements (lounge name/player number)]` command. 
- Added flag for "team points" for `/rr`. It adds an extra line to the race results that shows which placements each team got and their total points for that race. Usage example: `/rr 5 teampts=yes`

## Checklist

- [x] If code changes were made, they have been tested
- [ ] This PR fixes an issue
- [ ] This PR is not a code change (ie. documentation, typehinting, comments, etc.)

### CODE REVIEW

**Comment by BadWolf1023** [comment]
In response to the lounge name attribute, you are correct. It is not in use. It was planned in the original Table Bot design, but I'm not sure what happened. The only benefit to not using it is if someone's Lounge name changed mid-event - a fairly uncommon situation. It would be nice to use the attribute.

File: common.py [hunk 1]
```diff
@@ -290,7 +290,7 @@ def modify_property(modified_properties: Dict):
 
 #Rather than using the builtin set declaration {}, I did an iterable because BadWolfBot.py kept giving an error in Eclipse, even though everything ran fine - this seems to have suppressed the error which was giving me major OCD
 mkw_lounge_staff_roles = set([387347888935534593, #Boss
-                            #   792805904047276032, #CT Admin
+                              792805904047276032, #CT Admin
```
Comment (BadWolf1023) [inline common.py:293]: There's a CT Admin again?

File: Room.py [hunk 2]
```diff
@@ -567,10 +609,17 @@ def set_populating_miis(self, populating: bool) -> bool:
...
         return self.get_fc_to_name_dict().keys()
-    
-    def getPlayers(self):
-        return self.get_fc_to_name_dict().values()
 
+    @property
+    def players(self):
+        return self.getPlayers()
+
+    def getPlayers(self) -> List[Player]:
```
[the remark this answers is not in the document]
Comment (camelwater) [reply Room.py:617]: Yeah, room.players was never called before. (It's only called in ScoreKeeper.py)

File: ScoreKeeper.py [hunk 3]
```diff
@@ -198,41 +194,44 @@ def get_war_table_DCS(channel_bot:TableBot.ChannelBot, use_lounge_otherwise_mii=
     for x in range(numGPs):
         GPs.append(calculateGPScoresDCS(x+1, room, missingRacePts, server_id))
         
-    fcs_players = room.get_fc_to_name_dict(1, numGPs*4)
+    # fcs_players = room.get_fc_to_name_dict(1, numGPs*4)
+    players = room.players
```
[the remark this answers is not in the document]
Comment (camelwater) [reply ScoreKeeper.py:198]: You're right. I fixed it

File: common.py (hunk 1 above)
Comment (camelwater) [reply common.py:293]: No. I don't remember why I uncommented. I think it was an accident

File: Race.py [hunk 4]
```diff
@@ -389,6 +397,24 @@ def entireRoomBlankTimes(self):
     def multipleBlankTimes(self):
         return not self.entireRoomBlankTimes() and sum(1 for placement in self.getPlacements() if placement.is_disconnected()) > 1
         
+    def get_team_points_string(self, teams_data):
+        team_placements = defaultdict(list)
+        for placement in self.placements:
+            tag = teams_data[placement.player.FC]
+            team_placements[tag].append(placement.place)
+        
+        score_matrix = common.SCORE_MATRIX[len(self.placements)-1]
```
[the remark this answers is not in the document]
Comment (camelwater) [reply Race.py:406]: Fixed

File: api/endpoints.py [hunk 5]
```diff
@@ -31,13 +31,24 @@ def get_team_scores_html(
         text_color: Optional[str] = Query(None),
         font: Optional[str] = Query(None),
         border_color: Optional[str] = Query(None),
-        text_size: Optional[int] = Query(None)
+        text_size: Optional[int] = Query(None),
+        show_team_names: bool = Query(True),
```
[the remark this answers is not in the document]
Comment (camelwater) [reply api/endpoints.py:35]: Fixed

**Pull request merged**